### PR TITLE
WAZO-680 provd API version in URL

### PR DESCRIPTION
**Pull request by bloom1**

### CODE REVIEW

File: source/upgrade/upgrade_notes.rst [hunk 1]
```diff
@@ -12,6 +12,8 @@ Upgrade notes
   not backup or synchronized anymore.
 * The `/var/lib/xivo/sounds` directory has been migrated to `/var/lib/wazo/sounds` and the directory
   `/var/lib/xivo` is considered deprecated. Please update all custom references to this path.
+* `xivo-provisioning` API URL has been updated to remove the `provd` prefix and add the API version
+  number, which is `0.2`. All affected services and `wazo-provd-client` have been updated.
```
Comment (sduthil) [inline source/upgrade/upgrade_notes.rst]: Could you provide a "before" and "after" URL as an example?

**Comment by sduthil** [comment]
There are many references to `/provd/dev_mgr` (for example) left. Should we remove them in favor of http://developers.wazo.io/?

**Comment by bloom1** [comment]
> There are many references to `/provd/dev_mgr` (for example) left. Should we remove them in favor of http://developers.wazo.io/?

Considering that it is almost a copy of what the api.yml does, I think it would be better to take what is relevant from the current documentation and transfer it to api.yml. For example, there are some response descriptions that are not present in the api.yml.

File: source/upgrade/upgrade_notes.rst (hunk 1 above)
Comment (bloom1) [reply source/upgrade/upgrade_notes.rst]: Fixed

File: source/api_sdk/rest_api/provd/api.rst [hunk 2]
```diff
@@ -2,28 +2,7 @@
...
-
-.. toctree::
-   :maxdepth: 1
-
-   provd
-   devices
-   configs
-   plugins
-   general
+API documentation is available on http://api.wazo.community.
```
Comment (sduthil) [inline source/api_sdk/rest_api/provd/api.rst]: Could you remove the file altogether and the reference from the [Reference section](https://github.com/wazo-pbx/wazo-doc/blob/wazo-19.07/source/api_sdk/rest_api/reference.rst)?

Comment (bloom1) [reply source/api_sdk/rest_api/provd/api.rst]: Yes, it is indeed useless.

**Comment by sduthil** [comment]
Thank you!

**Pull request closed without merging**